index.js

```diff
@@ -51,6 +53,8 @@ export var utcMonths = utcMonth.range;
 export var utcYears = utcYear.range;
 
 export {
+  ticks as timeTicks,
+  tickInterval as timeTickInterval,
```

Comment:
Namespacing to avoid collision with ticks. I guess the problem solving this bottom up is I'm not 100% sure on the use cases for this API addition.